src/main/java/org/mskcc/cmo/messaging/impl/NATSGatewayImpl.java

```diff
@@ -46,16 +42,11 @@ public void publish(String topic, Object message) throws Exception {
     }
 
     @Override
-    public void subscribe(String topic, Class messageClass) throws Exception {
+    public void subscribe(String topic, MessageHandler messageHandler) throws Exception {
```

Comment:
You know what, I'm thinking this isn't a good change because it is exposing our use of the nats library.  Lets revert it back.